### ISSUE TITLE
Issue on page /pairwise/waterman.html

### ISSUE THREAD

**Issue by LegendenJoel** [statisticalbiotechnology/bibook#10]
In the figure text of Fig 5.3. If I'm not mistaken, you write "value" twice when trying to write "largest value corner"? If this is the case you should probably change the sentence somehow so that it makes sense. Either "... largest value, max ij S ij, corner ..." or "... largest value corner, max ij S ij, ..."

Samuel Emtell

**Comment by percolator** [comment]
Yes, that is a weird error that makes the text very hard to understand. Hope it reads better now!

Commit [af3ee6d](https://github.com/statisticalbiotechnology/bibook/commit/af3ee6daa38226f3ee7048f6ec06b1a4808612cb)
